test/e2e/main_test.go

```diff
@@ -250,8 +210,8 @@ func requireExtensionClean(t *testing.T, workDir string) {
 	err := os.Chdir(workDir)
 	require.NoError(t, err, `error changing to directory: %v`, workDir)
 
-	cmd := getEnvoy("extension clean")
```

Comment:
these cmd -> c changes were mainly to help avoid clashing with the package

Reply:
+1